components/server/src/auth/verification-service.ts

```diff
@@ -93,17 +97,45 @@ export class VerificationService {
             throw new ResponseError(ErrorCodes.INVALID_VALUE, "The given phone number is blocked due to abuse.");
         }
         const verification = await this.verifyService.verifications.create({ to: phoneNumber, channel });
-        log.info("Verification code sent", { phoneNumber, status: verification.status, channel });
+        log.info("Verification code sent", {
+            phoneNumber,
+            status: verification.status,
+            // actual channel verification was created on
+            channel: verification.channel,
+            // channel we requested - these could differ if a channel is not enabled in a specific country
+            requestedChannel: channel,
+        });
+
+        // Help us identify if verification codes are not able to send via requested channel
+        if (channel !== verification.channel) {
+            log.info("Verification code sent via different channel", {
```

Comment:
```suggestion
            log.info("Verification code sent via different channel than system requested", {
```
just for more context